groups/groups_test.go

```diff
@@ -25,25 +25,59 @@ import (
 	"strings"
 	"testing"
 	"unicode/utf8"
+
+	"k8s.io/apimachinery/pkg/util/sets"
 )
 
 var cfg GroupsConfig
 
-var groupsConfigPath = flag.String("groups-config", "./groups.yaml", "Path to groups config")
+var groupsConfigFileName = flag.String("groups-file", "groups.yaml", "Filename containing groups config")
```

Comment:
My personal preference would be to alter things to take an explicit `groups-path` or `groups-dir`, instead of auto-deriving it from the location of `config.yaml`

I also wonder if it would be useful to allow files named something other than `groups.yaml`, in the same way that prow jobs can be defined in arbitrarily named files.  Not blocking on this, just thinking out loud. I could foresee wg-k8s-infra using this to have a rbac file, a staging groups file, etc.

Reply:
> explicit groups-path or groups-dir, instead of auto-deriving it from the location of config.yaml

> I also wonder if it would be useful to allow files named something other than groups.yaml

This is interesting! Would something like "read all yaml files under the directory specified by `groups-dir`" make sense? (I still need to look at how prow does it today)